app/queue/queue.py

```diff
@@ -74,6 +74,8 @@ def _get_identity(host, metadata):
         if host.get("reporter") == "rhsm-conduit" and host.get("subscription_manager_id"):
             identity = deepcopy(SYSTEM_IDENTITY)
             identity["account_number"] = host.get("account")
+            if host.get("org_id"):
+                identity["org_id"] = host.get("org_id")
```

Comment:
Same as before, you can just use the `get()`.
```suggestion
            host.get("org_id")
```

Reply:
Done!